src/components/AppRouter/__tests__/AppRouter.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { mockLocation, loginSuccessMock, authContext } from '__mocks__';
+import { MemoryRouter } from 'react-router-dom';
+import { MockedProvider } from '@apollo/react-testing';
+import { ROUTES } from 'shared/routes';
+import { AuthContext } from 'context/AuthStore';
+import { UserRole } from 'types';
+import auth from 'utils/auth';
+import AppRouter from '../AppRouter';
+
+jest.mock('utils/auth');
+
+describe('Component - AppRouter', () => {
+  const unauthorisedContext = {
+    isAuthenticated: false,
+  };
+
+  const newbieContext = {
+    data: {
+      token: 'some_token',
+      role: UserRole.Newbie,
+      userId: '1',
+    },
+  };
+
+  const renderAppRouter = (context: any) =>
```

Comment:
I guess it was hard to use `AuthContextData` type here since context mocks doesn't fit AuthContext and `any` was just easy way to bypass TS errors?

Reply:
That's correct :eyes:, just for test I guess it's not so critical :poop: